src/backend/migrations/006-create-submission-photos-table.sql

```diff
@@ -0,0 +1,17 @@
+-- ## Migration to:
+-- * Create submission_photos table
+
+-- Start a transaction
+BEGIN;
+
+CREATE TABLE submission_photos (
```

Comment:
Check this SQL doesn't error if you run it twice 👍

Maybe `create table if not exists` will fix

Reply:
okay